src/util.ts

```diff
@@ -88,3 +89,19 @@ export function createOutputRetainingCompilerHost(
     outputFiles.set(fileName, content);
   }
 }
+
+/**
+ * True if we are running on Windows (32 or 64 bit).
+ */
+export const isWindows: boolean = os.platform() === 'win32';
+
+/**
+ * Returns the input with line endings normalized to '\n', when running on Windows.
+ * Returns the string as is, for any other OS.
+ */
+export function normalizeLineEndings(input: string): string {
+  if (isWindows)
```

Comment:
Is it important to only do so on Windows? Because if not, I'd just unconditionally replace `\r\n` with `\n`.